invoices_service.go

```diff
@@ -160,3 +160,23 @@ func (s *invoicesImpl) MarkFailed(invoiceNumber int) (*Response, *Invoice, error
 
 	return resp, &dst, err
 }
+
+// Allows custom invoice amounts to be refunded and generates a refund invoice.
+// Full open amount refunds of invoices with an unsettled transaction will void
+// the transaction and generate a void invoice.
+// https://dev.recurly.com/docs/line-item-refunds
+func (s *invoicesImpl) RefundVoidOpenAmount(invoiceNumber int, amountInCents int, refundApplyOrder string) (*Response, *Invoice, error) {
+	action := fmt.Sprintf("invoices/%d/refund", invoiceNumber)
+	req, err := s.client.newRequest("POST", action, nil, Invoice{
+		AmountInCents:    amountInCents,
+		RefundApplyOrder: refundApplyOrder,
+	})
```

Comment:
Updated in [e9b88a](https://github.com/blacklightcms/recurly/pull/11/commits/1b52bfba87d06be765b03b13c108236146e9b88a). Also made a comment about what's required for this call.